.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -1,8 +1,13 @@
-Description
-Types of change
-Checklist
+## Change Description
+<!-- Describe your changes and reference issues or pull requests where possible (use '#XX' where XX is the PR or issue number). -->
 
-    I have signed the CLA.
-    My code includes unit tests
-    I've added all required information.
-    My code doesn't require a change in the project's Protobuf definition (github.com/microsoft/presidio-genproto) or if they do, I've submitted a PR for a protobuf change.
+## Issue reference
+This PR fixes issue #XX
+
+## Checklist
+- [ ] I have reviewed the [contribution guidelines](../../CONTRIBUTING.md)
+- [ ] I have signed the CLA
+- [ ] My code includes unit tests
+- [ ] All unit tests and lint checks pass locally
+- [ ] My PR contains documentation updates / additions if required
+- [ ] My code doesn't require a change in the project's Protobuf definition (github.com/microsoft/presidio-genproto) or if they do, I have submitted a PR for a protobuf change.
```

Comment:
#YY protobuf pr link?

Reply:
You mean to ask them to provide the protobuf PR link in this pr?